app/views/checkout/thankyou.scala.html

```diff
@@ -36,6 +36,47 @@
         currency: '@subscription.currency',
         subscriptionId: '@subscription.id.get'
     };
+
+    function getCookie(name) {
+        var nameEQ = name + '=';
+        var ca = document.cookie.split(';');
+        for (var i = 0; i < ca.length; i++) {
+            var c = ca[i];
+            while (c.charAt(0) === ' ') { c = c.substring(1, c.length); }
+            if (c.indexOf(nameEQ) === 0) { return c.substring(nameEQ.length, c.length); }
+        }
+        return null;
+    }
+
+    function getAwinChannel() {
+        const channel = getCookie('gu_referrer_channel');
+        if (channel === null) {
+            return 'direct'; //No referrer the user came direct to the site
+        }
+
+        const values = channel ? channel.split('&') : null;
+        if (values && values[0] === 'afl' && values[1] === 'awin'){
+            return 'aw';
+        }
+        return 'na'; //A referrer other than Awin
+    }
+
+    //<![CDATA[ /*** Do not change ***/
+    var AWIN = {};
+    AWIN.Tracking = {};
+    AWIN.Tracking.Sale = {};
+
+    /*** Set your transaction parameters ***/
+    const productData = guardian.pageInfo.productData;
```

Comment:
Does subs have a list of supported browsers? Using `const` here will cause issues with, e.g. IE10. Is it worth using `var` here as with the `AWIN` variable just in case? I think the `const` is globally scoped anyway because it's sitting at the top level of a script tag.